<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginPage</class>
 <widget class="QMainWindow" name="loginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>621</width>
     <height>301</height>
    </size>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>_assets/logosingle.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>196</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtUser"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtPass">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnlogin">
         <property name="text">
          <string>Log in</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="errorlog">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSignup">
    <property name="geometry">
     <rect>
      <x>461</x>
      <y>200</y>
      <width>94</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShow">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;O&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnForgot">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>94</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Forgotten?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtUser</tabstop>
  <tabstop>txtPass</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnlogin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
